### PR TITLE
@eessex => [Publishing] Related Articles Canvas

### PR DESCRIPTION
**Pull request by kanaabe**
- Adds a RelatedArticlesCanvas
- Moves the AuthorDate component to the top-level /publishing dir
- New `related_articles` dir. For naming I went with Panel + Canvas, which maps similarly to the ad offerings.

Desktop:
image

Mobile:
image

### CODE REVIEW

File: src/components/publishing/related_articles/related_article_figure.tsx [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    </ArticleFigure>
+  )
+}
+
+const ImageTitle = styled.a`
+  display: flex;
+  flex-direction: column;
+  color: black;
+  text-decoration: none;
+  height: 270px;
```
Comment (eessex) [inline src/components/publishing/related_articles/related_article_figure.tsx]: It is possible that long titles could overflow this height?  Maybe min-height would make sense here -- not sure if the character count on thumbnail titles will cover it.

File: src/components/publishing/related_articles/related_articles_canvas.tsx [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+}
+
+const RelatedArticlesContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  max-width: 1250px;
+  margin: 30px auto 60px auto;
+  ${pMedia.lg`
+  margin: 30px 20px 60px 20px;
+  `}
```
Comment (eessex) [inline src/components/publishing/related_articles/related_articles_canvas.tsx]: * minor, indent missing in pMedia

**Comment by eessex** [comment]
Some very minor comments, feel free to address or merge

**Comment by kanaabe** [comment]
Addressed! Thanks. Will merge this after GH has its moment. Seems like they're having webhook delays.

**Comment by eessex** [comment]
Ahh, is that why circle is not running?

**Pull request merged**